src/pgwire/src/message.rs

```diff
@@ -406,6 +406,7 @@ impl ErrorResponse {
             CoordError::UnknownParameter(_) => SqlState::UNDEFINED_OBJECT,
             CoordError::UnknownPreparedStatement(_) => SqlState::UNDEFINED_PSTATEMENT,
             CoordError::UnknownLoginRole(_) => SqlState::INVALID_AUTHORIZATION_SPECIFICATION,
+            CoordError::UnknownClusterReplica { .. } => SqlState::FEATURE_NOT_SUPPORTED,
```

Comment:
This doesn't seem like the correct error, but I couldn't find a better one. Any thoughts @sploiselle?

Reply:
Agreed re: `UNDEFINED_OBJECT`